recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/system/stubdom-helpers.te

```diff
@@ -119,3 +126,48 @@ allow audio_helper_t self:shm create_shm_perms;
 init_dontaudit_use_fds(audio_helper_t)
 init_dontaudit_use_script_fds(audio_helper_t)
 xc_dontaudit_read_etc_files(audio_helper_t)
+
+########################################
+#
+# Local Policy: varstored
+#
+gen_require(`
+	type xen_device_t, xenstored_t, xenstored_var_run_t;
+')
+
+# varstored-watch executing & monitoring varstored
+allow varstored_t bin_t:dir search;
+allow varstored_t initrc_t:fd use;
+allow varstored_t initrc_t:unix_stream_socket connectto;
```

Comment:
This looks like the other end isn't labeled properly.  Or it should not be allowed.